### PR TITLE
Stop the autoclick - SellHouse.

### PR DESCRIPTION
**Pull request by CamaroZ**
Stop the autoclick in time to sell the house.

If someone does not know, when selling the house could use quick click when opening the dialog.

SORRY MY BAD ENGLISH...

### CODE REVIEW

File: Altis_Life.Altis/core/housing/fn_sellHouse.sqf [hunk 1]
```diff
@@ -19,6 +19,12 @@ closeDialog 0;
 _houseCfg = [(typeOf _house)] call life_fnc_houseConfig;
 if (count _houseCfg isEqualTo 0) exitWith {};
 
+if( player getVariable "sellinghouse"== 1) exitWith{closeDialog 0;};
```
Comment (sargken) [inline Altis_Life.Altis/core/housing/fn_sellHouse.sqf]: Your Indentation needs to be fixed to follow the coding guidelines.

Comment (CamaroZ) [inline Altis_Life.Altis/core/housing/fn_sellHouse.sqf]: Ok thx

File: Altis_Life.Altis/core/housing/fn_sellHouse.sqf [hunk 2]
```diff
@@ -19,6 +19,12 @@ closeDialog 0;
 _houseCfg = [(typeOf _house)] call life_fnc_houseConfig;
 if (count _houseCfg isEqualTo 0) exitWith {};
 
+if (player getVariable "selling" == true) exitWith{closeDialog 0;};
```
Comment (tkcjesse) [inline Altis_Life.Altis/core/housing/fn_sellHouse.sqf]: Uh... This is the same as line 24... ? Obviously it would never make it to line 24...

**Comment by sargken** [comment]
https://github.com/CamaroZ/Altis/pull/1

File: Altis_Life.Altis/core/housing/fn_sellHouse.sqf (hunk 2 above)
Comment (CamaroZ) [inline Altis_Life.Altis/core/housing/fn_sellHouse.sqf]: I did not see , sorry .

**Comment by BoGuu** [comment]
Please refer to https://community.bistudio.com/wiki/setVariable firstly.

Secondly, I have no idea what you plan to achieve with this, could you make an issue with perhaps a video to explain? You've added checks in odd places.

`
if (player getVariable "selling" == true) exitWith{closeDialog 0;};

if (player getVariable "selling" == true) exitWith {hint "Not fast.";};

player setVariable["selling",nil,true];
`

Will have a negligible time difference between them, making it next to pointless. Also, like I said, refer to the link, you are setting the selling variable to nil, undefining it.

**Pull request closed without merging**